Week1/Nadege_Awah/solution_to_palindrome_exercise.js

```diff
@@ -0,0 +1,15 @@
+//defines function reverseString and gives it a parameter str
+function reverseString(str) { 
```

Comment:
Also your solution will fail for a case such as `Racecar`

Reply:
I think Racecar is not a palindrome because the reverse gives racecaR. @DanCarl857 please I don't understand why a case like that should be considered.

Reply:
technically it's a palindrome because if you make it all lowercase you get a palindrome. 
So try making the entire string all lowercase or all uppercase before checking.

PS: Don't forget the mock interview session in a 2mins. Link on slack